packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -91,7 +92,8 @@ import { SyntheticEvent } from 'react';
 import { TeamsCall } from '@azure/communication-calling';
 import { TeamsCallAgent } from '@azure/communication-calling';
 import { TeamsMeetingLinkLocator } from '@azure/communication-calling';
-import { Theme } from '@fluentui/react';
+import { Theme as Theme_2 } from '@fluentui/react';
```

Comment:
I'm wondering if it's possible to improve the types in this file?

Reply:
It's auto-generated, so maybe in the future. There are lots of people on the api-extractor github asking questions about this.